client/app/telecom/pack/xdsl/access/comfortExchange/pack-xdsl-access-comfort-exchange.html

```diff
@@ -0,0 +1,127 @@
+<div>
+
+    <div class="widget-presentation">
+        <h2 class="widget-presentation-title" data-translate="xdsl_access_comfort_exchange_title">
+        </h2>
+
+        <tuc-toast-message></tuc-toast-message>
+
+        <div ng-if="$ctrl.isRMAOpened">
+            <div data-translate="xdsl_access_comfort_exchange_already_opened_rma"></div>
+            <br/>
```

Comment:
I don't think `br` is relevant here

It has the semantic meaning of a new line, for example when you are writing a poem or an adress

Use a margin instead